modules/signing_worker/manifests/settings.pp

```diff
@@ -0,0 +1,10 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class signing_worker::settings {
+
+    $virtualenv_dir = '/Users/cltbld/virtualenvs'
+    $user = 'cltbld'
+    $group = 'staff'
```

Comment:
Is this a leftover ? I couldn't find anywhere it's referenced, and `virtualenv_dir` looks like it wouldn't work in the dep case.

Reply:
I think it is, yes. I'll remove it.